azure-pipelines.yml

```diff
@@ -15,7 +15,7 @@ stages:
       - job: code_style_check
 
         pool:
-          vmImage: ubuntu-16.04
+          vmImage: ubuntu-20.04
```

Comment:
Have Azure actually removed `ubuntu-latest`?  The [docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#software) seem to say `ubuntu-latest` still exists. If they really have removed `ubuntu-latest` now, then we will also have to update `mbed-tools` again.